version/info.go

```diff
@@ -31,6 +31,7 @@ var (
 	BuildUser string
 	BuildDate string
 	GoVersion = runtime.Version()
+	Platform  = runtime.GOOS + "/" + runtime.GOARCH
```

Comment:
Two labels might be even better? After all, if you just want to select by Linux or architecture, that'll make things even easier then.

Reply:
+1, let's split this into separate fields.

Reply:
what would the fields be? `OS` and `Architecture`? Is it okay to leave it as one field in line 79 for the version information?